CONTRIBUTING.md

```diff
@@ -2,6 +2,52 @@
 
 Contact us on our Discord at [https://discord.gg/YjK2JAD](https://discord.gg/YjK2JAD)
 
-Feel free to make pull requests or issues, but please note that we are currently focused on making the mod, not adding more content.
+Feel free to make pull requests or issues, but please note that we are currently focused on making the mod functional, before adding more content.
 
-People who make pull requests should be familiar with the mod licenses; if you make a PR, you agree to the license.
+## PR Guidelines
+
+* Keep PR's to one topic, if possible. This permits us to use version control to find software regressions more easily.
+* Avoid mass operations such as folder moves and restructures next to logical changes in the code.
+* Use the PR message to describe changes; use markdown check marks to show intended progress towards a goal.
+* Keep PR up to date with the `master` branch to ease merges. In the future, once we have one, keep up to date with `develop` instead.
+* PR's should come with their code documented and unit tested (if possible).
```

Comment:
We reserve the right to push back if we disagree on whether it's possible.

Reply:
I think that is implied here. Obviously, if we don't like what we see, we don't merge it.

Reply:
I think adding the words will make contributors think a bit harder before deciding they can't test it.